tribe-ext-export-mec-events.php

```diff
@@ -0,0 +1,362 @@
+<?php
+/**
+ * Plugin Name:       The Events Calendar Extension: Export Events from Modern Events Calendar
+ * Plugin URI:        https://theeventscalendar.com/extensions/tribe-ext-export-mec-events
+ * Description:       Export Events from Modern Events Calendar
+ * Version:           1.0.0
+ * Extension Class:   Tribe__Extension__Export_Events_Modern_Events_Calendar
+ * GitHub Plugin URI: https://github.com/mt-support/tribe-ext-export-mec-events
+ * Author:            Modern Tribe, Inc.
+ * Author URI:        http://m.tri.be/1971
+ * License:           GPL version 3 or any later version
+ * License URI:       https://www.gnu.org/licenses/gpl-3.0.html
+ * Text Domain:       tribe-ext-export-mec-events
+ *
+ *     This plugin is free software: you can redistribute it and/or modify
+ *     it under the terms of the GNU General Public License as published by
+ *     the Free Software Foundation, either version 3 of the License, or
+ *     any later version.
+ *
+ *     This plugin is distributed in the hope that it will be useful,
+ *     but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *     GNU General Public License for more details.
+ */
+
+// Do not load unless Tribe Common is fully loaded and our class does not yet exist.
+if ( class_exists( 'Tribe__Extension' ) && ! class_exists( 'Tribe__Extension__Export_Events_Modern_Events_Calendar' ) ) {
+	/**
+	 * Extension main class, class begins loading on init() function.
+	 */
+	class Tribe__Extension__Export_Events_Modern_Events_Calendar extends Tribe__Extension {
+
+		/**
+		 * Setup the Extension's properties.
+		 *
+		 * This always executes even if the required plugins are not present.
+		 */
+		public function construct() {
+			$this->add_required_plugin( 'Tribe__Events__Main' );
+			$this->set_url( 'https://theeventscalendar.com/extensions/' );
+		}
+
+		/**
+		 * Extension initialization and hooks.
+		 */
+		public function init() {
+			add_action( 'admin_init', array( $this, 'add_admin_settings' ) );
+			add_action( 'load-tribe_events_page_' . Tribe__Settings::$parent_slug, array(
+				$this,
+				'listen_for_export_button',
+			), 10, 0 );
+		}
+
+		/**
+		 * Add the Admin Settings
+		 */
+		public function add_admin_settings() {
+			if ( ! class_exists( 'Tribe__Extension__Settings_Helper' ) ) {
+				require_once dirname( __FILE__ ) . '/src/Tribe/Settings_Helper.php';
+			}
+
+			$setting_helper = new Tribe__Settings_Helper();
+
+			$setting_helper->add_field( 'export-defaults-mec', array(
+				'type' => 'html',
+				'html' => '<h3 id="tribe-import-global-settings">' . esc_html__( 'Migration Tools - Modern Events Calendar', 'the-events-calendar' ) . '</h3>',
```

Comment:
Throughout: the text domain should be specific to this plugin. Good example of setting up the text domain and the naming convention we're using [can be found here](https://github.com/mt-support/tribe-ext-calendar-customizer/blob/master/index.php#L51).